PDSA_System/Server/DataBase/Create.sql

```diff
@@ -3,30 +3,34 @@ USE NordicDoor;
 
 CREATE TABLE IF NOT EXISTS Bruker
 (
-    AnsattNr    INTEGER      NOT NULL,
-    Fornavn     VARCHAR(50)  NOT NULL,
-    Etternavn   VARCHAR(50)  NOT NULL,
-    Email       VARCHAR(254) NOT NULL,
-    PassordHash VARCHAR(200) NOT NULL,
-    Opprettet   DATETIME DEFAULT CURRENT_TIMESTAMP,
+    AnsattNr    INTEGER                 NOT NULL,
+    Fornavn     VARCHAR(50)             NOT NULL,
+    Etternavn   VARCHAR(50)             NOT NULL,
+    Email       VARCHAR(254)            NOT NULL,
+    PassordHash VARCHAR(200)            NOT NULL,
+    Opprettet   DATETIME                DEFAULT CURRENT_TIMESTAMP,
     Rolle       VARCHAR(50),
-    LederId     INTEGER  DEFAULT NULL,
+    LederId     INTEGER                 DEFAULT NULL,
     KEY LederId (LederId),
     FOREIGN KEY (LederId) REFERENCES Bruker (AnsattNr),
     CONSTRAINT PK_Bruker PRIMARY KEY (AnsattNr)
 );
 
+DESC Bruker;
+
 CREATE TABLE IF NOT EXISTS Team
 (
     TeamId      INTEGER NOT NULL AUTO_INCREMENT,
-    TeamLederId INTEGER NOT NULL,
+    TeamLederId INTEGER NOT NULL DEFAULT 0,
```

Comment:
Jeg hadde ikke satt dette som default. Vi trenger ikke default, da et team skal alltid ha en eksisterende eier.

BrukerId 0 kommer aldri til å eksistere
```suggestion
    TeamLederId INTEGER NOT NULL,
```

Reply:
man får error dersom man gjerner default 0

Reply:
Men det er fordi `TeamLederId` skal defineres. Team har alltid en eier

Reply:
da må parameterene i team-controller-metoden endres

Reply:
Mener du `CreateTeam`?
https://github.com/IS-200-G7/nordicdoor/blob/3ff82f7f5f594d0baea79e87d44f20391e168565/PDSA_System/Server/Controllers/TeamController.cs#L52-L64
Den funksjonen henter allerede `TeamLederId` fra objektet man sender inn fra client side. Kan ikke se hvor problemet ligger, med mindre du tenkte på en annen endpoint?

Reply:
Kan se nærmere på det imorgen ?